src/parser/nit.sablecc3xx

```diff
@@ -574,7 +579,7 @@ expr_eq~nopar~nobra {-> expr}
 	| {:gt} expr_bitor~nopar~nobra gt :no [expr2]:expr_bitor~nopar~nobra
 	| {:ge} expr_bitor~nopar~nobra ge :no [expr2]:expr_bitor~nopar~nobra
 	| {:starship} expr_bitor~nopar~nobra starship :no [expr2]:expr_bitor~nopar~nobra
-	| {:isa} expr_bitor~nopar~nobra kwisa :no type~nobra
+	| {:isa} expr_bitor~nopar~nobra kwisa :no type_atom~nobra
```

Comment:
Instead, compound types should be allowed between parentheses. That way, we will be at least able to test the code that handle the semantics of that operator.